.github/actions/workflow-setup.yml

```diff
@@ -0,0 +1,139 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/SchemaStore/schemastore/master/src/schemas/json/github-action.json
```

Comment:
do you need
`
on: [workflow_call]
`
?

Reply:
I don't think so - this is an action, not a workflow, so it'll stay in the same execution path. If we did a callable workflow, you'd lose the steps that we're being carried out.

Reply:
Callable workflows are useful for other things though, like if you wanted to trigger a command done on a github runner using the gh cli. At least that's my understanding.

Reply:
ah, thanks. I see now this is using the Composite Actions feature instead of Reusable Workflows. The differences are kinda subtle since both use `uses:` to wire up different parts of the pipeline.

I was mainly checking because I didn't notice any checks running for this PR. Usually even if there are errors in the configuration of a workflow it doesn't show up in the PR's check list but there will be usually some trace of it in the actions tab.

Reply:
I'm pretty certain i've done something wrong. I'll figure it out.